src/System.CommandLine.Tests/ParserTests.MultipleArguments.cs

```diff
@@ -176,6 +176,36 @@ public void arity_ambiguities_can_be_differentiated_by_type_convertibility()
                       .BeEquivalentTo(new[] { "one", "two" },
                                       options => options.WithStrictOrdering());
             }
+
+            [Theory]
+            [InlineData(true)]
+            [InlineData(false)]
+            public void Multiple_arguments_can_be_specified_after_option(bool allowMultipleArguments)
```

Comment:
We typically align the test name to the scenario. There are two different scenarios here, which the use of `Theory` kind of hides. I also think the existing behavior (your `false` case) is already tested.

So I'd suggest only testing the new behavior, and letting the test name describe it explicitly.

Reply:
No problem, just pushed these changes.